response_operations_ui/controllers/reporting_units_controllers.py

```diff
@@ -43,3 +43,13 @@ def generate_new_enrolment_code(collection_exercise_id, ru_ref):
 
     logger.debug('Successfully generated new enrolment code', collection_exercise_id=collection_exercise_id, ru_ref=ru_ref)
     return response.json()
+
+def resend_verification_email(party_id):
+    logger.debug('Re-sending verification email', party_id=party_id)
+    url = f'{app.config["BACKSTAGE_API_URL"]}/v1/reporting-unit/resend-verification-email/{party_id}'
+    response = requests.post(url)
```

Comment:
Consider doing:
<code>
response = requests.post(url)
try:
    response.raise_for_status()
except HTTPError:
    logger.exception("Resending of verification email failed")
    raise ApiError(response)</code>

If for some reason in the future it did a redirect or gave a 201 instead then it won't raise an error.  At the minimum add the logger.exception line from my example, but checking for 200 or raise for status is up to you!

Reply:
If we're going to use this method we should do it consistently across the whole repo. I wouldn't change just this case